extension/data.ts

```diff
@@ -86,6 +86,17 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+// prettier-ignore
+const CHAR_CODES_BELOW_0x3000_USED_IN_WORDS_IN_DICTIONARY = [
```

Comment:
Let me know if I should name this to something else. I couldn't think of a good name, so I wrote a very long constant variable.

Also let me know if I should move this constant so that it's in the `RcxDict` class as a static constant so that it follows the convention of the other constants.

Reply:
I got this list of character codes below 0x3000 by running this code:

```ts
const uniqueWords = [
  ...new Set(
    rcxDict.wordDict
      .trim()
      .split("\n")
      .map((line) => line.split(" ")[0])
  ),
];
const charactersBelow0x3000InDictionaryWords = [
  ...new Set(
    uniqueWords.flatMap((word) =>
      [...word].filter((char) => char.charCodeAt(0) < 0x3000)
    )
  ),
];
console.log(
  "charactersBelow0x3000InDictionaryWords",
  charactersBelow0x3000InDictionaryWords
);
```